packages/react-app/src/hooks/useBounties.ts

```diff
@@ -31,11 +31,12 @@ export function useBounties(url: string): useBountiesResponse {
 }
 
 export function useBounty(id?: string | string[]): useBountyResponse {
-	/**
-   * Single bounty
-   */
-
-	const { data, error } = useSWR<BountyCollection, unknown>(`/api/bounties/${id}`, axiosFetcher);
+	const { data, error } = useSWR<BountyCollection, unknown>(
+		id
```

Comment:
when would `id` be null?

Reply:
On page load or server side (NextJS Fuckery)